xds/src/main/java/io/grpc/xds/GcpAuthenticationFilter.java

```diff
@@ -127,8 +128,6 @@ public ClientInterceptor buildClientInterceptor(FilterConfig config,
       @Nullable FilterConfig overrideConfig, ScheduledExecutorService scheduler) {
 
     ComputeEngineCredentials credentials = ComputeEngineCredentials.create();
-    LruCache<String, CallCredentials> callCredentialsCache =
-        new LruCache<>(((GcpAuthenticationConfig) config).getCacheSize());
```

Comment:
That was the only call to get the cache size. You are just ignoring the cache size now?